### ISSUE TITLE
[BUG] Broken with v4.04

### ISSUE THREAD

**Issue by leopirolo** [Aelto/tw3-random-encounters-reworked#204]
Hello,

I tried to install the mod with [the new update of The Witcher 3 v4.04](https://gamewave.fr/the-witcher-3-wild-hunt/the-witcher-3-wild-hunt-la-mise-a-jour-4-04-ajoute-de-nouveaux-elements-sur-la-version-switch/) but it seems that it's no longer compatible anymore.

**What it does**
Mod does not compile on launch.

**What it should do**
Mod should compile on launch.

Thanks for the amazing mod! Wish I could help.

**Comment by CriticalFoxes** [comment]
Can confirm the complication issue on 4.0.4.

`Error [content0]game\player\playerwitcher.ws(11781): Could not find function 'SetRadialPotionUpperTimer'
Error [content0]game\player\playerwitcher.ws(11787): Could not find function 'SetRadialPotionLowerTimer'

Warning [modsharedimports]engine\environment.ws(30): Global native function 'EnableDebugOverlayFilter' was not exported from C++ code.
Warning [modsharedimports]engine\environment.ws(32): Global native function 'EnableDebugPostProcess' was not exported from C++ code.
Warning [content0]engine\showflags.ws(11): Global native function 'DebugSetEShowFlag' was not exported from C++ code.`

I have no other mods installed, RER only on next-gen PC. 

Thanks.

**Comment by Aelto** [comment]
Hello, yes the mod was incompatible with the new version. The [`v3.3`](https://github.com/Aelto/tw3-random-encounters-reworked/releases/tag/v3.3) that i just released fixes it!

**Comment by leopirolo** [comment]
Hey! Thank you for the quick and great work!

I performed a clean installation from scratch, and it compiled successfully. :)
Once in the game, I received a confirmation message that the mod is working, but strangely, I can't find the mod menu in the options.
Did you change something by chance?

**Comment by Aelto** [comment]
Thanks for confirming the compilation success. Perhaps your filelists were reset during the update and you must add the RER menu in there again?

**Comment by leopirolo** [comment]
As a precautionary measure, I completely uninstalled the game, cleared the directory, reinstalled the game, and only installed the RER mod. Just to let you know:
- I installed the mod by unzipping the tw3-random-encounters-reworked.zip archive into the game's root folder. I didn't use the PS1 executable because it doesn't seem to work on my system (PowerShell launches and immediately stops).
- I made sure to merge the conflicting files using WitcherScriptMerger.

However, the result remains the same:
- The game launches successfully.
- Once in the game, a message confirms that the mod is installed correctly.
- The mod asks me which settings I want to use (Vanilla or Enhanced).

Nevertheless, there is still no sign of the RER menu in the options :(
